include/picongpu/fields/currentDeposition/EmZ/EmZ.hpp

```diff
@@ -165,18 +169,6 @@ namespace picongpu
                 pmacc::traits::StringProperty propList("name", "EmZ");
                 return propList;
             }
-
-
-            /** get normalized in cell particle position
-             *
-             * @param x position of the particle
-             * @param i shift of grid (only integral positions are allowed)
-             * @return in cell position
-             */
-            DINLINE float_X calc_InCellPos(const float_X x, const float_X i) const
```

Comment:
Why do you not add this method to `Base`? This would avoid adding a comment to the places where it is used.

Reply:
I find the method useless. Position already means in-cell posiion, both generally in PIConGPU and in context of existing current deposition code

Reply:
As i added a comment above, no need to return this function